selfdrive/boardd/boardd.cc

```diff
@@ -260,8 +261,8 @@ void can_recv_thread() {
     uint64_t cur_time = nanos_since_boot();
     int64_t remaining = next_frame_time - cur_time;
     if (remaining > 0){
-      useconds_t sleep = remaining / 1000;
-      usleep(sleep);
+      int sleep = remaining / 1000 / 1000;
+      util::sleep_for(sleep);
```

Comment:
Can we add a more precise sleep here? I'd like to have higher accuracy that 1ms. This is what drives the whole controls stack.